workspaces/github-actions/.changeset/afraid-planes-matter.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage-community/plugin-github-actions': patch
+---
+
+**BREAKING** Removed the `entityGitHubActionsCard` entirely as this card was mistakenly added as an entity card rather than content.
```

Comment:
```suggestion
The following changes where made to the alpha version of this plugin for the new frontend system:

Removed the `entityGitHubActionsCard` entirely as this card was mistakenly added as an entity card rather than content.
```

Realized we should provide context on these changes as they don't apply to all versions of the plugin.